tools/files/linux/init-script

```diff
@@ -58,26 +76,29 @@ case "$1" in
     if is_running; then
         echo -n "Stopping $name..."
         kill `get_pid`
-        for i in {1..10}
-        do
-            if ! is_running; then
-                break
-            fi
-
-            echo -n "."
-            sleep 1
-        done
+        wait_for_exit
         echo
 
         if is_running; then
-            echo "Not stopped; may still be shutting down or shutdown may have failed"
-            echo "Not stopped; may still be shutting down or shutdown may have failed" >> $stdout_log
-            exit 1
+            msg="Not stopped; may still be shutting down or shutdown may have failed, sending SIGKILL"
+            echo "$msg"
+            echo "$msg" >> $stdout_log
+            kill -9 `get_pid`
```

Comment:
So with these changes, it fails to stop with the default `SIGTERM` signal?  When running this process with `node index.js`, the TERM signal properly shuts down, so we should understand why that doesn't work rather than KILL (if what I've suggested is the case)

Reply:
> So with these changes, it fails to stop with the default SIGTERM signal

This is only invoked if SIGTERM fails to stop the process in 10 seconds

> When running this process with node index.js, the TERM signal properly shuts down, so we should understand why that doesn't work rather than KILL (if what I've suggested is the case)

Except that it does not always stop, as in the referenced issue #212 .  The 2 are not mutually exclusive, the root cause of the process refusing to stop should also be fixed.

I would argue that guaranteeing that `service logdna-agent stop` will always stop the process is a preferable strategy to always expecting `SIGTERM` to succeed (and causing production issues for some customers if it does not).

Reply:
Also, `systemd` is the current standard for init systems, the default configuration for `systemd` services is:
```
KillSignal=SIGTERM
FinalKillSignal=SIGKILL
SendSIGKILL=yes
```
Which this PR roughly replicates that behavior.

Ref: https://www.freedesktop.org/software/systemd/man/systemd.kill.html

Reply:
Yep, all that's good, and we agree that SIGKILL should be a fallback to use.  I'm happy to report that we've found the actual shutdown issue to be related to some timers that needed to be cleaned up on shutdown, and that's what's causing `SIGTERM` not to work.

We will need to open a separate PR to fix the underlying issue, but we think your PR is useful to allow a definite KILL of the process if need be.  We should have our fixes landed in the next few days, so feel free to pull those changes and continue with this PR to update the bash script.